src/networking/responses/products-response.ts

```diff
@@ -1,31 +1,33 @@
-export interface Price {
+export interface PriceResponse {
   amount: number;
   currency: string;
 }
 
-export interface PurchaseOptionPrice {
+export interface PurchaseOptionPhaseResponse {
   period_duration: string | null;
-  price: Price | null;
+  price: PriceResponse | null;
   cycle_count: number;
 }
 
-export interface PurchaseOption {
+export interface PurchaseOptionResponse {
   id: string;
 }
 
-export interface SubscriptionPurchaseOption extends PurchaseOption {
-  base_price: PurchaseOptionPrice;
-  trial: PurchaseOptionPrice | null;
+export interface SubscriptionPurchaseOptionResponse
+  extends PurchaseOptionResponse {
+  base_phase: PurchaseOptionPhaseResponse | null;
+  trial_phase: PurchaseOptionPhaseResponse | null;
 }
 
 export interface ProductResponse {
-  current_price: Price;
   identifier: string;
-  normal_period_duration: string;
```

Comment:
I removed these since I changed the product-level API to use the price and duration from the base phase of the default purchase option. We might need to bring them back once we add support for in-app products, but I think it's better to remove it for now.